lcls-twincat-pmps/Library/Library.plcproj

```diff
@@ -22,7 +22,6 @@
     <Description>Collection of PMPS functionality for LCLS PLCs</Description>
     <CombineIds>false</CombineIds>
     <Title>PMPS</Title>
-    <DocFormat>reStructuredText</DocFormat>
```

Comment:
Realized I previously changed this without following through. It might make better documentation someday, but presently it is just blocking any kind of docs my library could have when you open it in TC.